<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWindow</class>
 <widget class="QMainWindow" name="WeatherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{border-image: url(:/image/weather/weather_background.jpg);}
#btn_back{border-image: url(:/image/back.svg);}
QLabel{
   color:white;
font-family:微软雅黑;
}
#label_datetime{font-family:微软雅黑;
font-size:18pt;
}
#label_weather{
font-family:微软雅黑;
font-size:18pt;
font-weight:bold;
}
#btn_city{
color:white;
font-family:宋体;
background:transparent;
font-size:18pt;
font-weight:bold;
}
#label_temprature{
font-family:微软雅黑;
font-size:28pt;
font-weight:bold;
}
#label_date1,#label_temp1,#label_wea1{
color:yellow
}</string>
   </property>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetime">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>2025-04-17
16:54:05  周四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>69</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>220</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_city">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>成都</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temprature">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_date7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>85</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>17日
星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon7">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/weather/qing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>17℃~32℃</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wea7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>430</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>晴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
